pkg/apis/kops/validation/legacy.go

```diff
@@ -40,21 +40,15 @@ func ValidateCluster(c *kops.Cluster, strict bool) field.ErrorList {
 	fieldSpec := field.NewPath("spec")
 	allErrs := field.ErrorList{}
 
-	// kubernetesRelease is the version with only major & minor fields
-	// We initialize to an arbitrary value, preferably in the supported range,
-	// in case the value in c.Spec.KubernetesVersion is blank or unparseable.
-	kubernetesRelease := semver.Version{Major: 1, Minor: 15}
-
 	// KubernetesVersion
+	// This is one case we return error because a large part of the rest of the validation logic depend on a valid kubernetes version.
```

Comment:
```suggestion
	// This is one case we return the error because a large part of the rest of the validation logic depends on a valid kubernetes version.
```